docs/latest/user/proxy-observability.md

```diff
@@ -0,0 +1,133 @@
+# Proxy Observability
+
+Envoy Gateway provide observability for the ControlPlane and the underlying EnvoyProxy instances.
+This guide show you how to config proxy observability, includes metrics, logs, and traces.
+
+## Prerequisites
+
+Follow the steps from the [Quickstart Guide](quickstart.md) to install Envoy Gateway and the example manifest.
+Before proceeding, you should be able to query the example backend using HTTP.
+
+[FluentBit](https://fluentbit.io/) is used to collect logs from the EnvoyProxy instances and forward them to Loki. Install FluentBit:
+
+```shell
+helm repo add fluent https://fluent.github.io/helm-charts
+helm repo update
+helm upgrade --install fluent-bit fluent/fluent-bit -f https://raw.githubusercontent.com/envoyproxy/gateway/latest/examples/fluent-bit/helm-values.yaml -n monitoring --create-namespace --version 0.30.4
+```
+
+[Loki](https://grafana.com/oss/loki/) is used to store logs. Install Loki:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/envoyproxy/gateway/latest/examples/loki/loki.yaml -n monitoring 
+```
+
+[Tempo](https://grafana.com/oss/tempo/) is used to store traces. Install Tempo:
+
+```shell
+helm repo add grafana https://grafana.github.io/helm-charts
+helm repo update
+helm upgrade --install tempo grafana/tempo -f https://raw.githubusercontent.com/envoyproxy/gateway/latest/examples/tempo/helm-values.yaml -n monitoring --create-namespace --version 1.3.1
+```
+
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) offers a vendor-agnostic implementation of how to receive, process and export telemetry data. 
+Install OTel-Collector:
+
+```shell
+helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+helm repo update
+helm upgrade --install otel-collector open-telemetry/opentelemetry-collector -f https://raw.githubusercontent.com/envoyproxy/gateway/latest/examples/otel-collector/helm-values.yaml -n monitoring --create-namespace --version 0.60.0
+```
+
+Expose endpoints:
+
+```shell
+LOKI_IP=$(kubectl get svc loki -n monitoring -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+TEMPO_IP=$(kubectl get svc tempo -n monitoring -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+```
+
+## Metrics
+
+By default, Envoy Gateway doesn't expose metrics of the EnvoyProxy instances. 
+You can enable metrics by setting the `telemetry.metrics.prometheus` in the `EnvoyProxy` CRD.
+
+Expose prometheus metrics endpoints:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/envoyproxy/gateway/latest/examples/kubernetes/metric/prometheus.yaml
+```
+
+Verify metrics:
```

Comment:
can we add grafana and a stock envoy dashboard in here ?

Reply:
need more work and screenshot, it can be separated.

Reply:
cool lets track it with a GH issue

Reply:
raised https://github.com/envoyproxy/gateway/issues/1764 to track this